src/test/kotlin/uk/gov/dluhc/notificationsapi/client/ElectoralRegistrationOfficeManagementApiClientTest.kt

```diff
@@ -70,6 +71,7 @@ internal class ElectoralRegistrationOfficeManagementApiClientTest {
             given(clientResponse.bodyToMono(ElectoralRegistrationOfficeResponse::class.java)).willReturn(
                 Mono.just(eroResponse),
             )
+            given(clientResponse.statusCode()).willReturn(HttpStatus.OK)
```

Comment:
do we know why these changes are now required?

Reply:
Without it you get error :
   `Cannot invoke "org.springframework.http.HttpStatusCode.value()" because "code" is null`
when `apiClient.getElctoralRegistrationOfficeGssCodes(eroId)` is called
So I think when .bodyToMono is called in `apiClient.getElctoralRegistrationOfficeGssCodes(eroId)` it's no longer mocking a status code.

There's the same changes for similar tests in the Proxy Springboot upgrade https://github.com/communitiesuk/eip-ero-proxy-applications-api/pull/298/files#diff-a704b3d33ec07dc1247fa0d0979b388e500775a01fb7498fcf02cb035e0146f8